assets/openobserve/main.py

```
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+
+import sys
+import time
+import subprocess
```

Ruff (0.12.2)

`subprocess` imported but unused

Remove unused import: `subprocess`

(F401)